metricbeat/module/aerospike/_meta/config.yml

```diff
@@ -1,2 +1,4 @@
 - module: aerospike
+  metricsets: ["namespace"]
```

Comment:
The reason we left out the metricsets in the short config is to make sure we always have the default behaviour an it's not influence by the config. So if we add one more metricset to the defaults, a user will get it automatically without having to change the config.

Reply:
I see this point, what about having them commented?